string timestamp
float64 latitude
float64 longitude
float64 antenna_altitude
float64 horizontal_dil
uint8 num_sats
